Seohee/Lv2/괄호 변환.js

```diff
@@ -0,0 +1,54 @@
+// 올바른 괄호 문자열인지 체크
+function isCorrectBracket(str) {
+    const stack = [];
+    let open = 0;
+    let close = 0;
+    for (let i = 0; i < str.length; i++) {
+        if (str[i] === '(') {
+            stack.push('(');
+            open++;
+        } else {
+            stack.pop();
+            close++;
+        }
+    }
+    if (stack.length === 0 && open === close)
+        return true;
+    else
+        return false;
```

Comment:
```suggestion
    return stack.length === 0 && open === close;
```
return 하는 값이 조건문 안과 동일하다면 이렇게 표현할 수도 있습니다.